docs/guide/index.asciidoc

```diff
@@ -86,31 +75,28 @@ The following sections show how to get started quickly with Elastic APM on a loc
 
 First, https://www.elastic.co/downloads/apm/apm-server[download APM Server] for your operating system and extract the package.
 
-In a production environment,
-you would put APM Server on it's own machines,
+In a production environment you would put APM Server on it's own machines,
 similar to how you run Elasticsearch.
 You _can_ run it on the same machines as Elasticsearch,
 but this is not recommended,
 as the processes will be competing for resources.
 
-To start APM Server,
-run:
+To start APM Server run:
```

Comment:
Commas are usually recommended after introductory phrases of 4 or more words.

Reply:
"introductory phrase" means the ones starting with  `to, if, when, for, etc.`?

Reply:
Typically the introductory phrase is followed by a comma then a noun, but in this case the noun "you" is implied. TBH, I think your sentence is OK because it's easy to parse. I just wanted to mention the general rule because I wouldn't want you to spend time removing commas that are OK.